### PR TITLE
[StructuralMechanicsApplication] Changing wrapper get GP

### PR DESCRIPTION
**Pull request by loumalouomega**
Transition to #2194 

This changes solid and solid shell elements

### CODE REVIEW

File: applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp [hunk 1]
```diff
@@ -491,23 +491,31 @@ void BaseSolidElement::CalculateDampingMatrix(
 
 void BaseSolidElement::CalculateOnIntegrationPoints(
     const Variable<bool>& rVariable,
-    std::vector<bool>& rOutput,
+    std::vector<bool>& rValues,
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp]: Why do you change this?
In the baseclass it is `rOutput`

**Review by philbucher — DISMISSED** [review]
So just to understand correctly:
After this PR, i.e. in #2194, we have to 
- remove the wrapper-fcts of `GetValues...` that internally only call `CalculateValues...`
- rename the `GetValues...` that are not only wrapper-fcts to `CalculateValues...`
right

Consider my comment pls before merging

After this is merged can you then please do what I did in #2776 for the elems that are part of this PR? That would be most appreciated :)

File: applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp (hunk 1 above)
Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp]: Fuck, one sec

Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp]: Done

**Review by philbucher — APPROVED** [review]
Thanks!

**Pull request merged**